.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+node_js:
+  - '0.10'
+  - '4.4'
+
+services:
+  - mongodb
```

Comment:
It's not using any `mongodb` at the moment, but yes, would be good to keep in mind at the moment.